legacy/Dockerfile.kubectl-build-deploy-dind

```diff
@@ -19,6 +19,7 @@ COPY scripts /kubectl-build-deploy/scripts
 COPY helmcharts  /kubectl-build-deploy/helmcharts
 
 ENV DBAAS_OPERATOR_HTTP=dbaas.lagoon.svc:5000
+ENV DOCKER_HOST=docker-host.lagoon.svc
```

Comment:
this probably should be set in the root Dockerfile - I think this one isn't built anymore?

Reply:
oh